src/components/Display.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Display = ({ result = 0 }) => <h1>{result}</h1>;
```

Comment:
Hey, let us use defaultProps to set the default value of result to `'0'`
and let it be a string, look at what you wrote on line `7` the code is expecting a string.